packages/lucide-static/README.md

```diff
@@ -0,0 +1,162 @@
+# Lucide Static
+
+This package include the following lucide implementations:
+
+- All svg files
+- Javascript library containing strings of svgs.
+- Icon fonts
+- Svg sprite
+
+> What is lucide? Read it [here](https://github.com/lucide-icons/lucide#what-is-lucide).
+
+## Why lucide-static?
+
+This package is suitable for very specific use cases for example if you want to use icon fonts, svg sprites, normal svgs or Common.js Svg strings in your javascript project.
+
+> ⚠️ It is not recommended to use this package for svg sprites or icon fonts for web pages/applications, for prototyping it is ok. We recommend to bundlers for web applications to make sure you only bundle the used icons from this icon library (Threeshaking). Otherwise it will load all the icons, making you webpage loading slower. Threeshaking is only available in the packages: [lucide](https://github.com/lucide-icons/lucide/tree/master/packages/lucide), [lucide-react](https://github.com/lucide-icons/lucide/tree/master/packages/lucide-react), [lucide-vue](https://github.com/lucide-icons/lucide/tree/master/packages/lucide-vue), [lucide-vue-next](https://github.com/lucide-icons/lucide/tree/master/packages/lucide-vue-next), [lucide-angular](https://github.com/lucide-icons/lucide/tree/master/packages/lucide-angular), [lucide-preact](https://github.com/lucide-icons/lucide/tree/master/packages/lucide-preact)
+
+## Installation
+
+## Package Managers
+
+```sh
+yarn add lucide-static
+
+# or
+
+npm install lucide-static
+```
+
+### CDN
+
+``` html
+<!-- Svg File -->
+<img src="https://unpkg.com/lucide-static@latest/icons/home.svg">
+
+<!-- Icon Font, not recommended for production! -->
+<script src="https://unpkg.com/lucide-static@latest/font/Lucide.ttf"></script>
+
+<!-- Icon Sprite, not recommended for production! -->
+<img src="https://unpkg.com/lucide-static@latest/sprite.svg#home">
+```
+
+## Usage
+### SVG Files
+
+#### Svg file as image
+
+To use it in for example html:
+
+``` html
+<!-- Svg File -->
+<img src="~lucide/icons/home.svg">
+```
+
+``` css
+.home-icon {
+  background-image: url(~lucide/icons/home.svg)
+}
+```
+Make sure you have the correct webpack loaders to make this work. [url-loader](https://v4.webpack.js.org/loaders/url-loader/)
+
+#### Svg file Inline
+
+You can simply import each svg by targeting `lucide-static/icons/{icon-name}.svg`.
+To use svgs in your project you can for example use a [svg loader](https://v4.webpack.js.org/loaders/svg-inline-loader/).
+
+```js
+import arrowRightIcon from 'lucide-static/icons/arrow-right'
+
+// return string of a svg
+```
+
+### SVG Sprite
+
+```html
+<!-- Icon Sprite, not recommended for production! -->
+<img src="https://unpkg.com/lucide-static@latest/sprite.svg#home">
+
+<!-- or -->
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+>
+  <use
+    href="https://unpkg.com/lucide-static@latest/sprite.svg#alert-triangle"
+  />
+</svg>
+```
+
```

Comment:
```suggestion

If you'd prefer, you can use CSS to hold your base SVG properties
```css
.lucide-icon {
    width: 24px;
    height: 24px;
    stroke: currentColor;
    fill: none;
    stroke-width: 2;
    stroke-linecap: round;
    stroke-linejoin: round;
}
    ```
and update the svg as follows
```svg
<svg
  xmlns="http://www.w3.org/2000/svg"
  class="lucide-icon"
>
<use
    href="https://unpkg.com/lucide-static@latest/sprite.svg#alert-triangle"
  />
</svg>
    ```
```

Apologies on the formatting on the suggestion here - the closing code markers need to be un-indented - but doing so broke Github's suggestion